### PR TITLE
small updates to README.md

### PR DESCRIPTION
**Pull request by dealingwith**
* adds cmake dependency
* changed some inline code to code fenced blocks
* moved text into comments inside code blocks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,27 +3,37 @@ An interpreted, procedural, [stack-based](https://en.wikipedia.org/wiki/Stack-or
...
 ## How to build and run pringle
-1. clone this repoo
-2. `mkdir build`
-3. `cd build`
-4. configure CMake project `cmake ..`
-5. build the executable `cmake --build .`
-6. run the executable `./pringlelang`
-7. edit example.txt to your heart's content
 
+1. clone this repoo
```
Comment (jamespeilunli) [inline README.md]: small typo ("repoo" -> "repo")

File: README.md [hunk 2]
```diff
@@ -57,56 +71,60 @@ Currently there are three types of values that can be represented in pringle:
...
+
 ```
+# variable declaration and use example
 3 4 + var n 
 ```
-*assigns the value 7 to the variable n*
 
 ```
-n print
+# assigns the value 7 to the variable n
```
Comment (jamespeilunli) [inline README.md]: It would be more clear if this comment was in the previous code block with `3 4 + var n`

**Review by jamespeilunli — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -57,56 +69,60 @@ Currently there are three types of values that can be represented in pringle:
   - see integers
 
 ### Variables
+
 You must declare and assign a variable at the same time. The expression before the `var` function/keyword is the value of the variable and the identifier after `var` is the variable's name. Note that all variables are in the global scope.
-* variable declaration and use example: 
+
 ```
+# variable declaration and use example
 3 4 + var n
```
Comment (dealingwith) [inline README.md]:
```suggestion
3 4 + var n  # assigns the value 7 to the variable n
```

File: README.md [hunk 4]
```diff
@@ -57,56 +69,60 @@ Currently there are three types of values that can be represented in pringle:
...
+
 ```
+# variable declaration and use example
 3 4 + var n 
 ```
-*assigns the value 7 to the variable n*
 
 ```
-n print
+# assigns the value 7 to the variable n
```
Comment (dealingwith) [inline README.md]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -3,27 +3,35 @@ An interpreted, procedural, [stack-based](https://en.wikipedia.org/wiki/Stack-or
...
 ## How to build and run pringle
-1. clone this repoo
-2. `mkdir build`
-3. `cd build`
-4. configure CMake project `cmake ..`
-5. build the executable `cmake --build .`
-6. run the executable `./pringlelang`
-7. edit example.txt to your heart's content
 
+1. clone this repoo
```
Comment (dealingwith) [inline README.md]:
```suggestion
1. clone this repo
```

**Review by jamespeilunli — APPROVED** [review]

**Pull request merged**